### PR TITLE
LPS-124672 Add translation export to content pages

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
To test, do:
- Enable feature flag: `echo 'enabled=B"true"' > ${liferay_home}/osgi/configs/com.liferay.layout.admin.web.internal.configuration.FFLayoutTranslationConfiguration.config`
- Go to Product Menu > Site Builder > Pages
- Open the dropdown in any page, select "Export Translation"

**Note**: Modal is still not showing because the `layoutId` passed to the request is not correct.

### CODE REVIEW

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/MillerColumnsDisplayContext.java [hunk 1]
```diff
@@ -643,6 +643,8 @@ else if (layout.isTypeContent()) {
 					"label",
 					LanguageUtil.get(
 						_httpServletRequest, "export-for-translation")
+				).put(
+					"layoutId", layout.getLayoutId()
```
Comment (ambrinchaudhary) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/MillerColumnsDisplayContext.java:647]: Check this value, it is not correct and the fetch request is failing and preventing the modal to show up.

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/2026

**Pull request closed without merging**